src/components/composite/aspect/DashboardSectionAspects/DashboardSectionAspects.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import DashboardSection from '../../sections/DashboardSection/DashboardSection';
+import AspectCard, { AspectCardAspect } from '../../common/cards/AspectCard/AspectCard';
+import { CardLayoutContainer, CardLayoutItem } from '../../../core/CardLayoutContainer/CardLayoutContainer';
+
+interface DashboardSectionAspectsProps {
+  aspects: AspectCardAspect[];
+  aspectsCount?: number;
+  hubNameId?: string;
+  challengeNameId?: string;
+  opportunityNameId?: string;
+}
+
+const DashboardSectionAspects: FC<DashboardSectionAspectsProps> = ({ aspects, aspectsCount, ...parentEntityIds }) => {
+  const { t } = useTranslation();
+
+  const headerText = aspectsCount ? `${t('common.aspects')} (${aspectsCount})` : t('common.aspects');
```

Comment:
this won't display `0`.

Reply:
Well, this is intentional.
Should we display `Aspects (0)` on an empty Aspects card?

Reply:
We should have some indication that there are no aspects

Reply:
I'm used to having a thing like "no items" view in place of a list / table when it's empty. If we have such a thing, I can use it there. But of course I can also make a quick change and just show `Aspects (0)`. Should I stick with the latter?

Reply:
Yes, just display 0 for now.

Reply:
Looks good?

Reply:
Just from looking at the code, it seems to be that it won't show 0 on no aspects. It's either the label or label + count